src/components/devsupport/services/rtl/rtlFlexMap.ts

```diff
@@ -71,6 +71,7 @@ const FlexWrapMapper: RTLFlexMapper<string> = {
 /**
  * Matches FlexBox style properties that can affect on Layout depending on LTR/RTL mode corresponding Mappers
  */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const RtlFlexMap: Record<string, RTLFlexMapper<any>> = {
```

Comment:
Currently `RTLFlexMapper` use only `string` as type parameter. I think for now we can declare `string` as a generic parameter. If in future we will add some number or boolean transforms we can add corresponding type explicitly (e.g. `string` | `number`). I think we can easily avoid `any` here.